13-TeV-examples/uproot_python/HZZAnalysis.ipynb

```diff
@@ -399,25 +415,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
     "# cut on lepton charge\n",
     "# paper: \"selecting two pairs of isolated leptons, each of which is comprised of two leptons with the same flavour and opposite charge\"\n",
     "def cut_lep_charge(lep_charge):\n",
     "# throw away when sum of lepton charges is not equal to 0\n",
-    "# first lepton is [0], 2nd lepton is [1] etc\n",
-    "    return lep_charge[0] + lep_charge[1] + lep_charge[2] + lep_charge[3] != 0\n",
+    "# first lepton in each event is [:, 0], 2nd lepton is [:, 1] etc\n",
+    "    return lep_charge[:, 0] + lep_charge[:, 1] + lep_charge[:, 2] + lep_charge[:, 3] != 0\n",
```

Comment:
change to vectorized calculation - the syntax is the same as for 2D numpy arrays